tests/integration/pure_cairo_vm/cheatcodes/warp/warp_test.cairo

```diff
@@ -5,10 +5,37 @@ func test_warp(){
     local deployed_contract_address;
 
     %{
-        ids.deployed_contract_address = deploy_contract("main").contract_address
-        warp(123, ids.deployed_contract_address)
+        ids.deployed_contract_address = deploy_contract("./src/main.cairo").contract_address
+        warp(ids.deployed_contract_address, 123)
     %}
 
     assert_not_zero(deployed_contract_address);
+
+    local timestamp;
+    %{
+        ids.timestamp = call(ids.deployed_contract_address, "timestamp_getter")[0]
+    %}
+
+    assert timestamp = 123;
+    return ();
+}
+
+func test_warp_with_invoke(){
```

Comment:
Can we test it for depth 2? warp contract B, invoke contract A, A calls B, B asserts it's warped

Can be done in a separate PR